### PR TITLE
Add JIT label and label smaller subsets of tests

### PR DESCRIPTION
**Pull request by DrewScoggins**

### CODE REVIEW

File: src/benchmarks/micro/Categories.cs [hunk 1]
```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Data;
```
Comment (billwert) [inline src/benchmarks/micro/Categories.cs:5]: why?

Comment (DrewScoggins) [reply src/benchmarks/micro/Categories.cs:5]: I saw that as well. I have no idea how that got there.

File: src/benchmarks/micro/runtime/Bytemark/ByteMark.cs [hunk 2]
```diff
@@ -381,6 +382,7 @@ public void BenchNumericSortJagged()
     const int NumericSortRectangularIterations = 5;
 
     [Benchmark]
+    [BenchmarkCategory(Categories.JIT, Categories.ByteMark)]
```
Comment (kunalspathak) [inline src/benchmarks/micro/runtime/Bytemark/ByteMark.cs:385]: >BenchmarkCategory [](start = 5, length = 17)

In other places, I have seen this attribute at class level. Can we do the same for this file as well? Or do they have to be at method level?

Comment (DrewScoggins) [reply src/benchmarks/micro/runtime/Bytemark/ByteMark.cs:385]: As long as you can have class level annotations and also method level annotations and they are additive I think that moving it up the class level is fine. @adamsitnik

File: src/benchmarks/micro/libraries/System.Numerics.BitOperations/Perf_BitOperations.cs [hunk 3]
```diff
@@ -10,7 +10,7 @@
 
 namespace System.Numerics.Tests
 {
-    [BenchmarkCategory(Categories.Libraries, Categories.SIMD)]
+    [BenchmarkCategory(Categories.Libraries, Categories.SIMD, Categories.JIT)]
     public class Perf_BitOperations
```
Comment (kunalspathak) [inline src/benchmarks/micro/libraries/System.Numerics.BitOperations/Perf_BitOperations.cs:14]: can you also tag Roslyn\CscBench.cs with "JIT"?

Comment (DrewScoggins) [reply src/benchmarks/micro/libraries/System.Numerics.BitOperations/Perf_BitOperations.cs:14]: Good catch, I forgot to save this file when I pushed the other changes.

File: src/benchmarks/micro/Categories.cs [hunk 4]
```diff
@@ -21,15 +21,20 @@ public static class Categories
...
         /// </summary>
         public const string Runtime = "Runtime";
-            public const string BenchmarksGame = "BenchmarksGame";
-            public const string Benchstones = "Benchstones";
-                public const string BenchF = "BenchF";
-                public const string BenchI = "BenchI";
-            public const string Inlining = "Inlining";
-            public const string V8 = "V8";
-            public const string Perflab = "Perflab";
-            public const string Virtual = "Virtual";
```
Comment (adamsitnik) [inline src/benchmarks/micro/Categories.cs:31]: the goal of the spacing was to show category structure, but I can live without it ;)

File: src/benchmarks/micro/runtime/Bytemark/ByteMark.cs [hunk 5]
```diff
@@ -471,7 +471,6 @@ public void BenchAssignJagged()
     const int AssignRectangularIterations = 5;
 
     [Benchmark]
-    [BenchmarkCategory("TestTestTest")]
```
Comment (adamsitnik) [inline src/benchmarks/micro/runtime/Bytemark/ByteMark.cs:474]: great catch, I have no idea how it got into master branch

File: src/benchmarks/micro/Categories.cs [hunk 6]
```diff
@@ -21,15 +21,20 @@ public static class Categories
...
+        public const string BenchmarksGame = "BenchmarksGame";
+        public const string Benchstones = "Benchstones";
+        public const string BenchF = "BenchF";
+        public const string BenchI = "BenchI";
+        public const string Inlining = "Inlining";
+        public const string V8 = "V8";
+        public const string Perflab = "Perflab";
+        public const string Virtual = "Virtual";
+        public const string ByteMark = "ByteMark";
```
Comment (adamsitnik) [inline src/benchmarks/micro/Categories.cs:32]: the `ByteMark` category is used only by a single type. What is the purpose of adding it?

**Pull request merged**